README.md

```diff
@@ -13,9 +13,139 @@ limitations under the License.
 
 [![License](https://img.shields.io/badge/License-Apache_2.0-lightgrey.svg)](https://opensource.org/licenses/Apache-2.0)
 
-Triton Model Analyzer
-=====================
+# Triton Model Analyzer
 
-  **NOTE: You are currently on main-dev branch. This branch is tracking 
-  development for the new verssion of Model Analyzer and is not currently 
-  usable.**
+**NOTE: You are currently on main-dev branch. This branch is tracking 
+development for the new version of Model Analyzer and is not currently 
+usable.**
+
+Triton Model Analyzer is a CLI tool to help with better understanding of the
+compute and memory requirements of the Triton Inference Server models. These
+reports will help the user better understand the trade-offs in different
+configurations and choose a configuration that maximizes the performance of
+Triton Inference Server.
+
+The current release of the Triton Model Analyzer is 1.0.0 and corresponds to
```

Comment:
You will have to remove this line for now, since we haven't had a release. They can be added back after the actual release.